lib/bozo/packagers/nuget.rb

```diff
@@ -140,10 +144,11 @@ def files
 
   class LibraryPackage
 
-    def initialize(project, extfiles, nuget)
+    def initialize(project, extfiles, nuget, excluded_references = [])
```

Comment:
Minor thing: Do we want to name the parameter `excluded_references` for consistency sake?